apps/api/src/server/applications/case-admin-officer/case-admin-officer.routes.js

```diff
@@ -1,8 +1,8 @@
-import express from 'express';
+import { Router as createRouter } from 'express';
 import { asyncHandler } from '../../middleware/async-handler.js';
 import { getApplications } from './case-admin-officer.controller.js';
 
-const router = new express.Router();
```

Comment:
is there a reason for all these? is it related to "that problem" with the routes?

Reply:
Its that JSdoc issue. I was copying over what was done on Web